tests_e2e/orchestrator/lib/agent_test_suite_combinator.py

```diff
@@ -265,9 +266,10 @@ def create_environment_list(self, test_suites: List[str]) -> List[Dict[str, Any]
     def create_existing_vm_environment(self, test_suites: List[str]) -> Dict[str, Any]:
         loader = AgentTestLoader(test_suites, self.runbook.cloud)
 
+        location = self.runbook.locations.split(',')[0]
```

Comment:
here we expect a single location. rather than split, passing a list should be an error

Reply:
similar for vmss